DOL.WHD.Section14c.Web/src/modules/components/anchorLink/anchorLinkDirective.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.directive('anchorLink', function($anchorScroll, $document) {
+    'use strict';
+
+    return {
+      template: '<a href role="button" scrollTo={{anchor}} class="dol-skip-to-main">skip to main content</a>',
+      replace: false,
+      link: function(scope, element, attr) {
+        element.bind('click', function() {
+          $anchorScroll(attr.scrollto);
```

Comment:
Angular name normalization always bugs me.  Should this be `attr.scrollTo` like in the `template` definition, or does Angular normalize those names to all lowercase?

Normally I'd expect the `scrollTo` attribute to become `scroll-to` in actual markup, but I think that's just when you're defining the attribute in the controller scope?  Angular has too many ways of doing roughly the same things...  😛